integration_tests/integration/select-teams.spec.ts

```diff
@@ -25,7 +25,7 @@ context('Select teams', () => {
       .hint()
       .should(
         'contain',
-        'Select all the teams you currently allocate cases to. This includes teams you manage and teams you allocate to when on duty.'
+        'Select all the teams you currently allocate cases to.  You only need to pick teams that have probation practitioners in them.'
```

Comment:
do the e2e tests run locally after this change?n Perhaps we do not have coverage for this copy